readme.md

```diff
@@ -167,6 +168,7 @@ Each rule has emojis denoting:
 | [no-for-loop](docs/rules/no-for-loop.md) | Do not use a `for` loop that can be replaced with a `for-of` loop. | ✅ | 🔧 |  |
 | [no-hex-escape](docs/rules/no-hex-escape.md) | Enforce the use of Unicode escapes instead of hexadecimal escapes. | ✅ | 🔧 |  |
 | [no-instanceof-array](docs/rules/no-instanceof-array.md) | Require `Array.isArray()` instead of `instanceof Array`. | ✅ | 🔧 |  |
+| [no-invalid-remove-event-listener](docs/rules/no-invalid-remove-event-listener.md) | Prevent calling removeEventListener with the result of an expression | ✅ |  |  |
```

Comment:
The description here needs to be regenerated.

Reply:
Is this what you mean? Make this description match the one from `docs/rules/no-invalid-remove-event-listener.md`?

Reply:
Run `npm run generate-rules-table` and `npm run generate-usage-example`